examples/index.html

```diff
@@ -86,7 +87,8 @@
             var app = new PIXI.Application({
                 view: view,
                 width: stageWidth,
-                height: stageHeight
+                height: stageHeight,
+                backgroundColor:0xFF0000,
```

Comment:
Could you fix the filter bug related to alpha and revert the example background change? I would like it to be black for AlphaFilter #41

Reply:
OK. I use `backgroundColor:0x000000`. 
When developer want to use other color for testing , just modify it.
`modify` is easier than `add`. (less typing).